scripts/string_color.js

```diff
@@ -1,12 +1,16 @@
+function songToColour(song) {
+  return song.colour ?? stringToColour(song.feeling);
```

Comment:
TIL about the ?? ES2020 Nullish coalescing operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Let’s hope no one in FriRoll still uses IE 🙈